### DISCUSSION TITLE
Hoooorray: Two Million PyPI Downloads since 2016

### DISCUSSION THREAD

**Discussion by JorjMcKie** [pymupdf/PyMuPDF#821] [General]
Anyone have a prediction of the date 😎 ?

**Comment by JorjMcKie** [comment]
Accumulated downloads:
2021-01-11: **1.952.434**
2021-01-12: **1.955.692**
2021-01-13: **1.961.353**
2021-01-17: **1.970.568**
2021-01-18: **1.973.935**
2021-01-19: **1.976.976**
2021-01-20: **1.978.930**
2021-01-21: **1.981.440**
2021-01-22: **1.983.550**
2021-01-23: **1.985.109**
2021-01-23: **1.993.118**
2001-01-27: **2.001.103**

**Reply by JorjMcKie** [reply]
~My guess: **_2021-01-24_** is the day!~
With more data on 19Jan.: **2021-01-27**

**Reply by cbm755** [reply]
I'll vote for 2021-01-23.

**Reply by dummifiedme** [reply]
My Vote is for 2021-01-22 💯

**Comment by JorjMcKie** [comment]
@cbm755 / @dummifiedme  - Since you both participated here: On May 10th we've topped 3 million downloads (3.003.895) 🎉🎈😎

This a million downloads within 3.5 months (since January 27).

Thank you for ongoing constructive support!

**Reply by cbm755** [reply]
... and probably less than half of those are automated downloads from me fighting with my downstream CI scripts! :smile: :partying_face:

**Reply by JorjMcKie** [reply]
😂 a well-deserved cold shower ...!
